src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -2051,6 +2051,7 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
 
             case NI_Sve_GatherVector:
             case NI_Sve_GatherVectorByteZeroExtend:
+            case NI_Sve_GatherVectorFirstFaulting:
```

Comment:
this should be handled similar to how we do `NI_Sve_LoadVectorFirstFaulting`.

Reply:
There is also lowering code that needs to be done for this intrinsic.

Reply:
also, this is not correct to reuse the code for `NI_Sve_GatherVector*` ....they have different instructions.